core/src/main/java/discord4j/core/object/entity/Member.java

```diff
@@ -314,6 +314,45 @@ public String getNicknameMention() {
                 .flatMap(ignored -> Mono.just(PermissionSet.all()))
                 .switchIfEmpty(Mono.zip(getEveryonePerms, getRolePerms, PermissionUtil::computeBasePermissions));
     }
+    
+    /**
+	 * Requests to determine if this member is higher in the role hierarchy than member.
+	 * This is determined by the positions of each of the members' highest roles.
+	 *
+	 * @param member The member who must be lower in the role hierarchy.
```

Comment:
Doesn't necessarily have to be a member lower in the role hierarchy. It's just a member to do a check against. So I would just say `The member to compare in the role hierarchy with this member.` Or something along those lines.